### ISSUE TITLE
revH_compat - `asp_cmnd.py` fails when etcd fills up

### ISSUE THREAD

**Issue by jaycedowell** [lwa-project/analog_signal_processor#9]
From the logs:
```
2023-09-27 06:04:21 [WARNING ] Could not get channel config. for board 8251: <_InactiveRpcError of RPC that terminated with:
        status = StatusCode.RESOURCE_EXHAUSTED
        details = "etcdserver: mvcc: database space exceeded"
        debug_error_string = "{"created":"@1695794661.429931894","description":"Error received from peer ipv4:127.0.0.1:2379","file":"src/core/lib/surface/call.cc","file_line":966,"grpc_message":"etcdserver: mvcc: database space exceeded","grpc_status":8}"
>
```

Blowing away the current etcd database fixes this but it will happen again.

**Comment by jaycedowell** [comment]
@ctaylor-physics you should probably be away that this can happen.

**Comment by jaycedowell** [comment]
This seems to have happened again.

**Comment by jaycedowell** [comment]
I tried this:
```
import etcd3

c = etcd3.Etcd3Client('127.0.0.1')

r = 1e12
for k in c.get_all():
    if k[1].mod_revision < r:
        r = k[1].mod_revision
r = (r//100)*100

c.compact(r, physical=True)
```

The database size, i.e., `c.status().db_size`, didn't show a huge change.  Adding in a `c.defragment()` after the compaction lead to a much smaller database size but INI still doesn't work.

**Comment by jaycedowell** [comment]
Added `compactEtcd.py` to run every Sunday to root's crontab.

**Comment by jaycedowell** [comment]
Fixed with #13.
